### DISCUSSION TITLE
Where to find latest edition of the IBM Style Guide

### DISCUSSION THREAD

**Discussion by emteelb** [redhat-documentation/vale-at-red-hat#391] [General]
Where is the latest version of the IBM Style Guide that the vale-at-red-hat project uses available?

I'm confused because the edition that I purchased says "First Printing: October 2011 with corrections March 2015." I checked with an IBM sales rep today and the rep said 2015 was the latest edition.

However, in the word usage appendix in my version, the `hostname` entry says:

```
Use only as a variable. Otherwise, use "host name."
```

Yet `vale-at-red-hat` flags occurrences of "host name" with a warning suggesting "hostname" instead.

I can see that the hostname/host name topic was addressed in a couple of issues. One [here](https://github.com/redhat-documentation/vale-at-red-hat/issues/145) that makes mention of a hostname entry on pg. 292 of the IBM Style Guide. However, pg. 292 in my version of the IBM Style Guide is a page within the DITA tags appendix.

**Comment by daobrien** [comment]
Sounds like you're referring to a hard copy, which hasn't been updated in quite a while. We now use the online version available at https://www.ibm.com/docs/en/ibm-style

You can log in with your RH email address. I think the first time you try to log in you'll be prompted to set up a password. afaik everyone with a RH address has access.

The current guidance is as follows:
host name   noun
Use with caution. Write as shown only when updating existing content that uses the two-word spelling. For new content, use "hostname".

hostname   noun
Write as shown unless updating existing content that uses "host name".

Hope this helps.

**Reply by emteelb** [reply]
Yes, helpful. Thanks for this info. A couple of weeks ago, I somehow found the link that you mentioned and requested access (through a link on the page) but haven't heard back yet from anyone at IBM. I don't have an RH email address, although the organization I'm with does have a collaborative development relationship with RH.

If you can suggest any other channels to go through to request access, would appreciate it.

**Reply by daobrien** [reply]
That could make it tricky. My only other suggestion at this point is to get in touch with Shannon Lebedev (slebedev@us.ibm.com), who helped me organize access for RH. Good luck.

**Comment by silopolis** [comment]
Le mer. 30 nov. 2022 à 17:38, Michael Troutman ***@***.***> a
écrit :

> Yes, helpful. Thanks for this info. A couple of weeks ago, I somehow found
> the link that you mentioned and requested access (through a link on the
> page) but haven't heard back yet from anyone at IBM. I don't have an RH
> email address

Same here :/

As mentioned in another comment, it would be awesome and a huge contribution from IBM to make that guide publicly available...
AFAIC, not being able to access it may be an issue for the guides' translation project I'm considering.
